src/main/zencode/stdlib/src_old/Strings.zs

```diff
@@ -0,0 +1,108 @@
+[Native("stdlib::String")]
```

Comment:
📔 Note: This file should contains all methods that are failing in Strings.zs.
It should be removed eventually.